retriever/lib/engine_tools.py

```diff
@@ -162,6 +164,25 @@ def xml2csv(input_file, outputfile=None, header_values=None, row_tag="row"):
     return outputfile
 
 
+def xlsxcsv(book):
+    workbook = xlrd.open_workbook(book)
+    dire = book.rstrip(".xlsx")
+    if not os.path.exists(dire):
+        os.makedirs(dire)
+    else:
```

Comment:
Is this `else` required?

Reply:
I was using it just to remove path already exists error, I'll remove it by adding `exist_ok=True`